cli/check_config_test.go

```diff
@@ -0,0 +1,17 @@
+package cli
+
+import (
+	"testing"
+)
+
+func TestCheckConfig(t *testing.T) {
+	err := CheckConfig([]string{"testdata/conf.good.yml"})
+	if err != nil {
+		t.Fatalf("error: %v", err)
+	}
+
+	err = CheckConfig([]string{"testdata/conf.bad.yml"})
+	if err == nil {
+		t.Fatalf("Failed to detect missing file")
```

Comment:
I am confused by the word `missing` here. How about `failed to detect invalid file`?

Reply:
Forgot to change this when I changed the test